cdc/kv/matcher.go

```diff
@@ -97,3 +108,19 @@ func (m *matcher) matchCachedRow() []*cdcpb.Event_Row {
 func (m *matcher) rollbackRow(row *cdcpb.Event_Row) {
 	delete(m.unmatchedValue, newMatchKey(row))
 }
+
+func (m *matcher) cacheRollbackRow(row *cdcpb.Event_Row) {
+	m.cachedRollback = append(m.cachedRollback, row)
+}
+
+func (m *matcher) matchCachedRollbackRow(initialized bool) {
+	if !initialized {
```

Comment:
why need this parameter since the initialized = false would never be called?

Reply:
No if there is no bug, it's an additional check to make sure we does not misuse `matchCachedRollbackRow` and `state.initialized`.